survival_guide/3_operations.adoc

```diff
@@ -677,39 +694,51 @@ With a access/secret pair, you’ll need to add the secret:
 [source,yaml]
 ----
 s3:
-    enableS3: true
-    enableIAM: false
-    accessKeyName: aws-access-key
-    secretKeyName: aws-secret-key
-    logDirectory: s3a://[BUCKET-NAME]
+  enableS3: true
+  enableIAM: false
+  accessKeyName: aws-access-key
+  secretKeyName: aws-secret-key
+  logDirectory: s3a://[BUCKET_NAME]
 ----
 
 ===== Configuring Spark
 
-After the History Server has been set up, then the Fusion job-launcher deployment ConfigMap’s `application.yaml` key will need to be updated with these Spark settings so the driver and executors know where to write out their logs. In this example, we’re redirecting to a GCS bucket:
+After the History Server has been set up, then the config map for Fusion's job-launcher will need its `application.yaml` key will need to be updated with these Spark settings so the driver and executors know where to write out their logs.
+
+In this example, having installed Fusion into a namespace of `sparkhistory`, we will edit the config map to write the logs to a Google Cloud Storage bucket.
+
+[source,bash]
+----
+kubectl edit cm spark-history-job-launcher
+----
+
+Update the `spark` key with these settings and then restart the `job-launcher` pod.
+
 
 [source,yaml]
 ----
 spark:
-    eventLog:
-        enabled: true
-        compress: true
-        dir: gs://[BUCKET-NAME]
-    hadoop:
-        google:
-            cloud:
-                auth:
-                    service:
-                        account:
-                            json:
-                                keyfile: /etc/secrets/[SECRET_KEY_NAME].json]
-    kubernetes:
-        driver:
-            secrets:
-                history-secrets: /etc/secrets
-        executor:
-            secrets:
-                history-secrets: /etc/secrets
+  …
+  eventLog:
+    enabled: true
+    compress: true
+    dir: gs://[BUCKET_NAME]
+  hadoop:
+    google:
+      cloud:
+        auth:
+          service:
+            account:
+              json:
+                keyfile: /etc/secrets/[ACCOUNT_NAME].json
+  kubernetes:
+    driver:
+      secrets:
+        history-secrets: /etc/secrets
+    executor:
+      secrets:
+        history-secrets: /etc/secrets
+  …
 ----
 
```

Comment:
We need to add:
1. How to go verify that it's working - run a job and check the history server.
2. How to actually reach and expose the spark history server.